src/main/java/de/dennisguse/opentracks/ControllerFragment.java

```diff
@@ -114,6 +118,12 @@ public void onResume() {
         trackRecordingServiceConnection.startConnection(getContext());
```

Comment:
Also Nullcheck this?

Reply:
not necessary as this will be created via onCreate.
The NPE's are happening in the delayed callbacks.